server/pages/index.jade

```diff
@@ -1,12 +1,12 @@
 doctype html
 //
 	<3
-	             _
-	    ___ __ _| |_   _ _ __  ___  ___
+				 _
```

Comment:
What changed here?

Reply:
FIxed! I converted spaces in the logo to tabs by mistake